missouri/demographics.yml

```
@@ -0,0 +1,117 @@
+demographics:
+  white:
+    ppl: 0.562
+    sex_type:
+      MSM:
+        ppl: 1.0
+        sex_role:
+          init:
+            versatile: 0.380
+            insertive: 0.356
+            receptive: 0.264
+        safe_sex:
+          prob: 0.234
+        prep:
+          discontinue: 0.125
+          adherence: 0.75
+          init: 0.03
+        drug_type:
+          None:
+            ppl: 1.0
+            hiv:
+              init: 0.280
+              dx:
+                init: 0.881
+                prob: 0.063
+              aids:
+                init: 0.496
+            haart:
+              init: 0.708
+              enroll: # unsure of start and stop times
```

Start and stop times are [start, stop), meaning the start time is included and stop time is not. So this would look like
```suggestion
              enroll:
                enroll_1:
                  start: 0
                  stop: 2
                  prob: 0.579
                enroll_2:
                  start: 2
                  stop: 12
                  ...
```